Corgibytes.Freshli.Cli/Commands/RunnableCommand.cs

```diff
@@ -0,0 +1,25 @@
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.CommandLine.NamingConventionBinder;
+using Corgibytes.Freshli.Cli.CommandRunners;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Corgibytes.Freshli.Cli.Commands;
+
+#nullable enable
+public abstract class RunnableCommand<T> : Command where T: CommandOptions.CommandOptions
+{
+    protected RunnableCommand(string name, string? description) : base(name, description)
```

Comment:
I don't see `description` being set to null in this project so far. And if it was, I don't think it would look awesome to see `null` passed in as a parameter.

I think it makes sense to either drop `?` from `string?` for `description` or to provide a default value of `null` so that it can be optionally provided.

So either:

```suggestion
    protected RunnableCommand(string name, string description) : base(name, description)
```

or:

```suggestion
    protected RunnableCommand(string name, string? description = null) : base(name, description)
```